package.json

```diff
@@ -69,5 +69,9 @@
         "webpack-dev-server": "^3.7.2",
         "webpack-merge": "^4.2.1"
     },
-    "proxy": "http://localhost:7200"
+    "proxy": "http://localhost:7200",
```

Comment:
Hm, I though that we had removed the proxy.

Reply:
No we don't.